{"timestamp":"2025-07-31T04:32:53.611284","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-31T04:32:53.611803","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/SFTP-dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-31T04:32:53.787141","level":"info","event":"Connection Retrieved 'dest_postgres'","logger":"airflow.hooks.base"}
{"timestamp":"2025-07-31T04:32:53.794699","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"UndefinedTable","exc_value":"relation \"file_data\" does not exist\nLINE 2:                     INSERT INTO file_data (filename, col1, c...\n                                        ^\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":877,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1164,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":217,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":240,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/SFTP-dag.py","lineno":61,"name":"load_to_postgres"}],"is_group":false,"exceptions":[]}]}
